Zappy_IA_Report/NN/Code/Tutorial4.py

```
@@ -0,0 +1,31 @@
+import tensorflow as td
+from tensorflow import keras
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)